llmfoundry/__init__.py

```diff
@@ -4,6 +4,26 @@
 import torch
 
 try:
+    import warnings
+
+    # bitsandbytes is a very noisy library. A lot of it is print statements that we can't easily suppress,
```

Comment:
Can you open an issue in their repo for this?

Reply:
sure, its a grad student project with 500 open issues tho

Reply:
https://github.com/TimDettmers/bitsandbytes/issues/919

Reply:
loll, thanks! hopefully a good first task for bits and bytes newcomers...? 😛